resources/assets/components/CampaignSingle/index.js

```diff
@@ -181,7 +181,12 @@ class CampaignSingle extends React.Component {
         </div>
         <FilterBar onSubmit={this.filterPosts}>
           <DropdownFilter options={statusFilters} header={'Post Status'} />
-          <MultiValueFilter options={tagFilters} header={'Tags'} />
+          <MultiValueFilter options={tagFilters}>
+            <b>Tags</b> Not sure which tag to filter by?{' '}
+            <a className="font-normal" href="/faq#tags">
+              See tag definitions.
+            </a>
```

Comment:
Is this text inside the `MultiValueFilter` what ends up as `this.props.children` in `MultiValueFilter/index.js`?

Reply:
I think this is a perfect way to handle this! ⚛️